ui/src/routes/[repository]/file/[fileVersion]/history/+page.svelte

```diff
@@ -45,7 +45,7 @@
                 <span class="timestamp">
                     {toTimestamp(fileVersion.id)}
 
-                    <input type=checkbox disabled={fileVersion.deleted || ['png', 'jpg', 'gif'].includes(data.fileVersion.fileName.split('.')[1])} bind:group={$diffIds} value={fileVersion.id}>
+                    <input type=checkbox disabled={fileVersion.deleted || ['png', 'jpg', 'gif', 'pdf'].includes(data.fileVersion.fileName.split('.')[1])} bind:group={$diffIds} value={fileVersion.id}>
```

Comment:
We should use .at(-1) instead, so files like file.name.png could work too.
also in the other places...

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/at

Reply:
solved here: https://github.com/DlieBG/gum/pull/47/commits/238e1b4ca1e10d39fa94dbf222f077ca374886c6